sw/device/lib/dif/dif_otp_ctrl.h

```diff
@@ -357,11 +357,39 @@ dif_result_t dif_otp_ctrl_check_integrity(const dif_otp_ctrl_t *otp);
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_otp_ctrl_check_consistency(const dif_otp_ctrl_t *otp);
 
+/**
+ * Locks out access to the direct access interface registers.
+ *
+ * This function is reentrant: calling it while functionality is locked will
```

Comment:
Perhaps you mean idempotent, as in once locked it stays locked.

Reply:
Yeah I can reword this (this wording is used in other places of this DIF as well).